__tests__/utils/operations.js

```diff
@@ -545,4 +545,17 @@ module.exports = {
     )
     return groupsJson
   },
+  async addGroupMembers(config, groupMembers) {
+    const result = await runInEnvironment(
+      function(configJSON, groupMembersJSON) {
+        var config = Tozny.storage.Config.fromObject(configJSON)
+        var client = new Tozny.storage.Client(config)
+        groupMembers = new Tozny.types.GroupMember(groupMembersJSON)
```

Comment:
is groupMember able to parse an array of groupMembers correctly? my understanding was that it only sets up one groupMember at a time; however, i haven't run your code yet, so it's very possible that this does actually work.

Reply:
Currently attempting to test that, i think you are right tho!

Reply:
I had a similar issue with listGroups. I think you can do something similar to what you're doing to decode the members in client.js (line 1229).